card/mod/machines/lib/javascript/decko/doubleclick.js.coffee

```diff
@@ -0,0 +1,27 @@
+doubleClickActiveMap = { off: false, on: true, signed_in: decko.currentUserId }
+
+doubleClickActive = () ->
+  doubleClickActiveMap[decko.doubleClick]
+  # else alert "illegal configuration: " + decko.doubleClick
+
+doubleClickApplies = (el) ->
+  return false if ['.nodblclick', '.d0-card-header', '.card-editor', '.bridge-sidebar'].some (klass) ->
```

Comment:
added bridge-sidebar

Reply:
why not add nodblclick to the bridge sidebar?  the other classes occur in several places, but I think the bridge-sidebar only appears once (or twice?), no?

More importantly, this may be tricky after we add guides in the bridge-sidebar. You had the good idea of making it so that edit text was double-clickable.  But the guide will be inside the bridge-sidebar.

My best guess is that we'll want to make this smarter, so that, eg, you can configure something clickable inside something not clickable, and inner configs overrule outer ones.